collector/monitors/monitors.go

```diff
@@ -9,8 +9,10 @@ type Monitor interface {
 	InitMetrics()
 	// Handler fetches the data to inner storage
 	Handler(ctx context.Context) error
-	// GetMetrics - provides metrics fetched by Handler method
+	// GetMetrics - provides single value metrics fetched by Handler method
 	GetMetrics() map[Metric]float64
+	// GetMetricVectors - provides set of a labeled values fetched by Handler method
+	GetMetricVectors() map[Metric]map[string]float64
```

Comment:
`map[Metric]map[string]float64` looks a bit cumbersome, maybe we could add a subtype?

Reply:
declared new type
 type MetricVector map[string]float64